src/components/ui/card.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react"
```

Comment:
Don't import whole package, just the things that you need. This can lead to really poor performance.